modular_skyrat/modules/company_imports/code/armament_datums/nakamura_modsuits.dm

```diff
@@ -0,0 +1,345 @@
+
+/datum/armament_entry/company_import/nakamura_modsuits/utility_modules/retract_plates
+	item_type = /obj/item/mod/module/plate_compression
+	lower_cost = CARGO_CRATE_VALUE * MODULE_MID_LOWER
+	upper_cost = CARGO_CRATE_VALUE * MODULE_MID_UPPER
+	interest_required = PASSED_INTEREST
+
+/datum/armament_entry/company_import/nakamura_modsuits/utility_modules/magnetic_deploy
+	item_type = /obj/item/mod/module/springlock/contractor
+	lower_cost = CARGO_CRATE_VALUE * MODULE_PRICEY_LOWER
+	upper_cost = CARGO_CRATE_VALUE * MODULE_PRICEY_UPPER
+	interest_required = HIGH_INTEREST
+	interest_addition = COMPANY_INTEREST_GAIN_AVERAGE
+
+// Mobility modules, jetpacks and stuff
+
+/datum/armament_entry/company_import/nakamura_modsuits/mobility_modules
+	subcategory = "MOD Mobility Modules"
+
+/datum/armament_entry/company_import/nakamura_modsuits/mobility_modules/tether
+	item_type = /obj/item/mod/module/tether
+	lower_cost = CARGO_CRATE_VALUE * MODULE_CHEAP_LOWER
+	upper_cost = CARGO_CRATE_VALUE * MODULE_CHEAP_UPPER
+	interest_addition = COMPANY_INTEREST_GAIN_PITIFUL
+
+/datum/armament_entry/company_import/nakamura_modsuits/mobility_modules/magboot
+	item_type = /obj/item/mod/module/magboot
+	lower_cost = CARGO_CRATE_VALUE * MODULE_CHEAP_LOWER
+	upper_cost = CARGO_CRATE_VALUE * MODULE_CHEAP_UPPER
+	interest_addition = COMPANY_INTEREST_GAIN_PITIFUL
+
+/datum/armament_entry/company_import/nakamura_modsuits/mobility_modules/jetpack
+	item_type = /obj/item/mod/module/jetpack
+	lower_cost = CARGO_CRATE_VALUE * MODULE_MID_LOWER
+	upper_cost = CARGO_CRATE_VALUE * MODULE_MID_UPPER
+	interest_required = PASSED_INTEREST
+
+/datum/armament_entry/company_import/nakamura_modsuits/mobility_modules/pathfinder
+	item_type = /obj/item/mod/module/pathfinder
+	lower_cost = CARGO_CRATE_VALUE * MODULE_MID_LOWER
+	upper_cost = CARGO_CRATE_VALUE * MODULE_MID_UPPER
+	interest_required = PASSED_INTEREST
+
+/datum/armament_entry/company_import/nakamura_modsuits/mobility_modules/disposals
+	item_type = /obj/item/mod/module/disposal_connector
+	lower_cost = CARGO_CRATE_VALUE * MODULE_MID_LOWER
+	upper_cost = CARGO_CRATE_VALUE * MODULE_MID_UPPER
+	interest_required = PASSED_INTEREST
+
+/datum/armament_entry/company_import/nakamura_modsuits/mobility_modules/sphere
+	item_type = /obj/item/mod/module/sphere_transform
+	lower_cost = CARGO_CRATE_VALUE * MODULE_PRICEY_LOWER
+	upper_cost = CARGO_CRATE_VALUE * MODULE_PRICEY_UPPER
+	interest_required = HIGH_INTEREST
+	interest_addition = COMPANY_INTEREST_GAIN_AVERAGE
+
+/datum/armament_entry/company_import/nakamura_modsuits/mobility_modules/atrocinator
+	item_type = /obj/item/mod/module/atrocinator
+	lower_cost = CARGO_CRATE_VALUE * MODULE_PRICEY_LOWER
+	upper_cost = CARGO_CRATE_VALUE * MODULE_PRICEY_UPPER
+	interest_required = HIGH_INTEREST
+	interest_addition = COMPANY_INTEREST_GAIN_AVERAGE
+
+/datum/armament_entry/company_import/nakamura_modsuits/mobility_modules/super_jetpack
+	item_type = /obj/item/mod/module/jetpack/advanced
```

Comment:
not sold on having this, veryvery strong

Reply:
removed